src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -99,6 +99,12 @@ public Encoding Encoding
 #endif
 
         /// <summary>
+        /// Gets or sets a value indicating whether to auto-check if the console is available
+        /// </summary>
```

Comment:
No idea, but I can extend the summary a bit. Seldom I dare to use any properties in NLog without reading the code first :)